services/vpm/vpm-version.service.js

```diff
@@ -0,0 +1,81 @@
+import Joi from 'joi'
+import { optionalUrl } from '../validators.js'
+import { latest, renderVersionBadge } from '../version.js'
+import { BaseJsonService, NotFound } from '../index.js'
+
+const queryParamSchema = Joi.object({
+  repositoryUrl: optionalUrl.required(),
```

Comment:
Can we make the query param `repository_url`. To make the variable name `repositoryUrl` internally, you can change the function signature for `handle()` to

```js
async handle({ packageId }, { repository_url: repositoryUrl, sort }) {
```

(you'll need to update the examples to use `repository_url`)